src/Engine/ProtoAssociative/CodeGen.cs

```diff
@@ -453,7 +453,7 @@ private void UpdateType (string ident, int funcIndex, ProtoCore.Type dataType)
             ProtoCore.Type datatype, 
             int datasize = ProtoCore.DSASM.Constants.kPrimitiveSize, 
             bool isStatic = false,
-            ProtoCore.Compiler.AccessSpecifier access = ProtoCore.Compiler.AccessSpecifier.kPublic,
```

Comment:
@ke-yu  Sure, ill place these in the enhancements task. This is so we can make the changes smaller
http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6193